Index: rtl8821cs-wifi/Makefile
===================================================================
--- rtl8821cs-wifi.orig/Makefile
+++ rtl8821cs-wifi/Makefile
@@ -143,9 +143,9 @@ endif
 
 ########################## Debug ###########################
 CONFIG_RTW_DEBUG = y
-# default log level is _DRV_INFO_ = 4,
+# default log level is _DRV_WARN_ = 3,
 # please refer to "How_to_set_driver_debug_log_level.doc" to set the available level.
-CONFIG_RTW_LOG_LEVEL = 4
+CONFIG_RTW_LOG_LEVEL = 3
 
 # enable /proc/net/rtlxxxx/ debug interfaces
 CONFIG_PROC_DEBUG = y
